packages/options/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "display": "@solana/options",
+    "extends": "tsconfig/base.json",
+    "include": ["src"],
+    "compilerOptions": {
+        "lib": ["DOM", "ES2020", "ES2022.Error"]
```

Comment:
Copypasta? Eliminate items from this array until typechecking fails, and then put back only that which is needed to make it pass.